journal/modernized/33-testimonies.adoc

```diff
@@ -369,8 +362,9 @@ He was very helpful and strengthening to Friends in those times of great calamit
 he was also concerned in addressing the government and chiefest men
 in authority on behalf of Friends and the English inhabitants,
 and they commonly would hear him,
-and often granted relief He was careful in advising Friends,
+and often granted relief. He was careful in advising Friends,
 that they should not touch with any goods, where property was dubious, in those times;
+//"touch with any goods" is underlined and questioned by the proofreader, but again I can't find the PDF :(
```

Comment:
This appears to be an error. Its in some of the google PDFs this way, and in another version it was changed to _"that they should not touch any goods where property was dubious"._  Lets switch it to the version I have here.